src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -283,6 +283,10 @@ export default {
 			:disabled="disabled"
 			:type="inputType"
 			:value="value"
+			:checked="isChecked"
+			:indeterminate.prop="indeterminate"
+			:required="required"
+			:name="name"
 			v-bind="inputProps"
```

Comment:
`v-bind="inputProps"` should be removed then

Reply:
Oop! Ty!